src/components/Nav/Nav.js

```diff
@@ -0,0 +1,13 @@
+import {NavLink} from 'react-router-dom';
+import '../StarWars/Films/film.css';
+
+export const Nav = () => (
+  <div className="nav">
+    <NavLink className="filmBtn" exact activeStyle={{color: 'red'}} to='/'>Главная</NavLink>
```

Comment:
better use active class. For example filmBtnActive and then in css add  .filmBtnActive {  color: red }